packages/terra-action-header/src/ActionHeaderContainer.scss

```diff
@@ -0,0 +1,63 @@
+$default-border: 1px solid #1c1f21;
+
+:local {
+  .flex-header {
+    background-color: var(--terra-action-header-background-color, #fff);
+    border-bottom: var(--terra-action-header-border-bottom, $default-border);
+    border-left: var(--terra-action-header-border-left, $default-border);
+    border-right: var(--terra-action-header-border-right, $default-border);
+    border-top: var(--terra-action-header-border-top, $default-border);
+    box-shadow: var(--terra-action-header-box-shadow);
+    display: flex;
+    justify-content: space-between;
+    padding: var(--terra-action-header-padding, 0.714rem);
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-fill {
+    align-self: center;
+    flex: 1 1 auto;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-end {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
+    height: 100%;
+    position: relative;
+  }
+
+  .flex-collapse {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
+    height: 100%;
+    max-width: 40%;
+    position: relative;
+  }
+
+  .title-container {
+    height: 100%;
+    left: 0;
+    position: relative;
+    top: 0;
+    width: 100%;
+  }
+
+  .title {
+    color: var(--terra-action-header-font-color, #1c1f21);
+    font-family: var(--terra-action-header-font-family);
```

Comment:
`font-family` can be deleted. It will be inherited via the terra-base styles which have a themable font-family.

Reply:
font-family removed [here](https://github.com/cerner/terra-core/commit/85b35f2f932e64ced0e37458cd33fde2f0633fe7).